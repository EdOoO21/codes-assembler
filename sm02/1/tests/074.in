55 776 -768 -347 -753 638 -367 -628 792 753 912 -557 -281 -755 -96 641 -65 73 892 -431 -87 -651 -359 -429 29 279 -994 411 53 -910 -740 937 684 -533 -103 -682 952 223 -140 -519 -449 -478 359 787 -106 -1000 128 -586 761 358 896 -674 160 -884 507 -556 466 881 -317 -913 -959 -963 413 730 955 919 -784 -190 419 160 12 469 -190 -702 428 -651 226 -643
