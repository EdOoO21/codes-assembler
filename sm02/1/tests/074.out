-643
226
-651
428
-702
-190
469
12
160
419
-190
-784
919
955
730
413
-963
-959
-913
-317
881
466
-556
507
-884
160
-674
896
358
761
-586
128
-1000
-106
787
359
-478
-449
-519
-140
223
952
-682
-103
-533
684
937
-740
-910
53
411
-994
279
29
-429
-359
-651
-87
-431
892
73
-65
641
-96
-755
-281
-557
912
753
792
-628
-367
638
-753
-347
-768
776
55
